packages/@aws-cdk/aws-dynamodb/README.md

```diff
@@ -87,6 +87,7 @@ times of the day:
 Auto-scaling is only relevant for tables with the billing mode, PROVISIONED.
 
 [Example of configuring autoscaling](test/integ.autoscaling.lit.ts)
+[Example of configuring the table as a scalable target](test/integ.dynamodb.scalable-target.ts)
```

Comment:
The above example is a `.lit.ts` example, which is a legacy way of providing code examples in readme. It actually [shows up as code](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_dynamodb-readme.html#configure-autoscaling-for-your-table) when rendered in the docs. Pretty cool!

However, 1) your example would not be rendered, and 2) we discourage doing examples like this anymore. Lets just put an example into markdown without linking to the integ test.